deployment/v2/allow-partial-builds.sh

```diff
@@ -37,6 +37,7 @@ then
   echo 'Error - You must run this script from the aws-solutions-constructs folder.'
   echo
 else
+  npm install -g aws-cdk
```

Comment:
Yes - the new test tool, integ-runner, shells out to the CDK. Since we've switched, we've always had to manually install the CDK in a new docker container (and constantly forget). This will install once, perhaps reinstall with a later version if the container is long-lived, but still is harmless.